tools/notifyServicelogEvent.c

```diff
@@ -81,14 +86,27 @@ static pcmk__cli_option_t long_options[] = {
     },
     { 0, 0, 0, 0 }
 };
+*/
+
+static GOptionContext *
+build_arg_context(pcmk__common_args_t *args, GOptionGroup **group)
+{
+    GOptionContext *context = NULL;
+    const char *description = "event_id is a unique unsigned event identifier "
+                              "that is passed into servicelog.\n";
+
+    context = pcmk__build_arg_context(args, NULL, NULL, "event_id");
+    g_option_context_set_description(context, description);
+
+    return context;
+}
 
 int
 main(int argc, char *argv[])
 {
-    int argerr = 0;
-    int flag;
-    int index = 0;
-
+    //int argerr = 0;
+    //int flag;
+    //int index = 0;
```

Comment:
whoops

Reply:
Latest update removes this and a bunch of other commented out code that was left in this commit.